tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeGhQuantities.cpp

```diff
@@ -626,6 +629,115 @@ void test_spacetime_derivative_of_spacetime_metric(
   }
 }
 
+// Test GeneralizedHarmonic::ricci_tensor
+void test_spatial_ricci_tensor(
```

Comment:
I think this utility is well-suited to a pypp test -- that would probably be easier to maintain in future than the hard-coded values from SpEC.

Reply:
Added a random python test too, I think we can retain the test using hard-coded values from spec since it already exists + its unlikely that we'll be modifying this code as it is for a very specific expression.